pom.xml

```diff
@@ -39,8 +39,8 @@
     </modules>
 
 	<properties>
-		<openMRSVersion>1.9.9</openMRSVersion>
-		<openMRSMinorVersion>1.9</openMRSMinorVersion>
+		<openMRSVersion>1.10.5</openMRSVersion>
+		<openMRSMinorVersion>1.10</openMRSMinorVersion>
 		<htmlwidgetsVersion>1.7.2</htmlwidgetsVersion>
```

Comment:
What we probably want to do is to locate all of this in a new reporting-1.10 module, which is conditionally loaded (similar to the 2.0 module) if 1.10+ is running.